OsmAnd/res/xml/monitoring_settings.xml

```diff
@@ -124,8 +124,6 @@
 	<net.osmand.plus.settings.preferences.SwitchPreferenceEx
```

Comment:
Do we still need RECORD_OBD_DATA preference? or is it only UI pref now & we can change SwitchPreferenceEx to Preference?